src/components/BulkEnrollmentResultsDownloadPage/index.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useContext, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Redirect, useParams } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { logError } from '@edx/frontend-platform/logging';
+import { camelCaseObject } from '@edx/frontend-platform';
+import { ToastsContext } from '../Toasts';
+import EnterpriseAppSkeleton from '../EnterpriseApp/EnterpriseAppSkeleton';
+import LicenseManagerApiService from '../../data/services/LicenseManagerAPIService';
+
+const BulkEnrollmentResultsDownloadPage = ({ enterpriseId }) => {
```

Comment:
it's not specific to this PR but I would love for us to start using a pages/ vs components/ pattern at some point. Seems like we have a lot of components some of them pages, and it's unclear which one plays what role sometimes. However, we can do sth slightly different to organize the pages for the time being, if you are interested, we can actually have a src/components/bulkenrollment/ folder at least in there, and have the two pages be in there?

Reply:
Sounds good - I am still pretty green with React so you'll have to lead the way in defining the conventions.